client/dashboard/components/code-highlighter/index.tsx

```diff
@@ -0,0 +1,30 @@
+import Prism from 'prismjs';
+import { useEffect, useRef } from 'react';
+import 'prismjs/themes/prism.css';
+import 'prismjs/components/prism-yaml';
+
+import './style.scss';
+
+interface CodeHighlighterProps {
+	content: string;
+}
+
+export const CodeHighlighter = ( { content }: CodeHighlighterProps ) => {
+	const yamlCodeRef = useRef< HTMLElement | null >( null );
+
+	useEffect( () => {
+		if ( yamlCodeRef.current ) {
+			Prism.highlightElement( yamlCodeRef.current );
+		}
+	}, [ content ] );
+
+	return (
+		<>
```

Comment:
nit: do we need the outer fragment here

Reply:
Good point, removed in 3e4ffa044214b465f27e34ae28a8226a1fef0091